feature-libs/product-configurator/rulebased/occ/cpq/index.ts

```diff
@@ -1,2 +1,4 @@
+export * from './converters/index';
 export * from './cpq-configurator-occ.module';
-export { defaultOccCpqConfiguratorConfigFactory as ɵdefaultOccCpqConfiguratorConfigFactory } from './default-occ-cpq-configurator-config';
+export * from './cpq-configurator-occ.service';
+export { defaultOccCpqConfiguratorConfigFactory as ɵdefaultOccCpqConfiguratorConfigFactory } from './default-occ-configurator-cpq-config';
```

Comment:
We don't need the ɵ exports any longer. After we removed the exports from the very root entry point, our entities are exposed only once, in one entry point, this is also true for the implicit exports.
So the stuff we don't intend to expose we can just remove from index.ts